Examples/Tests/PythonWrappers/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,291 @@
+    vmin = -vmax
+    return (vmin, vmax)
+
+# Plot fields data either in valid domain or in PML
+def plot_data(data, pml, title, name):
+    fig, ax = plt.subplots(nrows = 1, ncols = 1, gridspec_kw = dict(wspace = 0.5), figsize = [6,5])
+    cax = make_axes_locatable(ax).append_axes('right', size='5%', pad='5%')
+    lw  = 0.8
+    ls = '--'
+    if (pml):
+        # Draw PMLs and ghost regions
+        ax.axvline(x = 0       , linewidth = lw, linestyle = ls)
+        ax.axvline(x = 0+nxg   , linewidth = lw, linestyle = ls)
+        ax.axvline(x = -nxpml  , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx      , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx-nxg  , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx+nxpml, linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0       , linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0+nzg   , linewidth = lw, linestyle = ls)
+        ax.axhline(y = -nzpml  , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz      , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz-nzg  , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz+nzpml, linewidth = lw, linestyle = ls)
+        # Annotations
+        ax.annotate('PML', xy = (-nxpml//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (nx+nxpml//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (nx//2,-nzpml//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (nx//2,nz+nzpml//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (-nxpml-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx+nxpml+nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,-nzpml-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz+nzpml+nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        # Set extent and sliced data
+        extent = np.array([-nxg-nxpml, nx+nxpml+nxg, -nzg-nzpml, nz+nzpml+nzg])
+    else:
+        # Draw ghost regions
+        ax.axvline(x = 0       , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx      , linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0       , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz      , linewidth = lw, linestyle = ls)
```

Comment:
White spaces left over:
```suggestion
        ax.axvline(x = 0 , linewidth = lw, linestyle = ls)
        ax.axvline(x = nx, linewidth = lw, linestyle = ls)
        ax.axhline(y = 0 , linewidth = lw, linestyle = ls)
        ax.axhline(y = nz, linewidth = lw, linestyle = ls)
```